go/private/actions/stdlib.bzl

```diff
@@ -38,12 +42,11 @@ def emit_stdlib(go):
     Otherwise, the standard library will be compiled for the target.
 
     Returns:
-        A list of providers containing GoLibrary, GoSource and GoStdLib.
+        A list of providers containing GoSource and GoStdLib.
     """
-    library = go.new_library(go)
-    source = go.library_to_source(go, {}, library, False)
+    source = new_source(go, {}, coverage_instrumented = False)
```

Comment:
i removed this helper from the context since it seemed simpler to just import the code we want to call, but I can put it back if desired